uwblueprint/website-bp
# Extend Application Deadline

### Implementation description

- Extend Application Deadlines.



### Steps to test

1. Ensure the new application deadline reflect on the website.
2. Ensure students can still apply.



### What should reviewers focus on?

- The new extension date is reflected.
- Applications can still be submitted.

### Checklist

- [ ] My PR name is descriptive and in the imperative tense
